app/views/pages/home.slim

```diff
@@ -8,7 +8,7 @@ section.phr.bg-white.tiber
               | Financial Intelligence for Cryptocurrencies
           .col-xs-12.col-md-5.col-lg-12
             .pl4-m.pl0-l.pr5-l.tc.pv3.pt0-m.pb0-l
-              img.max-w15e.max-w18e-s.max-w50e-m.w-100 src=image_path('ethereum-machine.png')
+              img.max-w15e.max-w18e-s.max-w50e-m.w-100 src=image_path('ethereum-machine.jpg')
```

Comment:
I wonder how much faster this page would load if we were using webp. is there a scaleable way this can be done across the app, rather than manually?